src/assets/i18n/en.json

```diff
@@ -1,8 +1,8 @@
 {
-    "ABOUT_DNS_TEXT": "Most people who use the internet have never reflected over what actually happens when one types an address in their web browser and then clicks the enter button. To simplify, the domain name system, (DNS) is a database that associates domain names to IP addresses, the same way a phone book associates names to phone numbers. An IP address is a unique series of numbers that identifies every computer that is connected to the internet. It’s similar to the way every telephone has its own number in the telephone network. A long series of numbers works fine for computers, but it’s difficult for people to memorize. Therefore, DNS has been developed an extra system for us when we want to reach a web server on a specific site or an email server we want to send an email to. ",
-    "ABOUT_ZONEMASTER_TEXT": "Give your domain a complete physical! Zonemaster helps you to control how your website is doing. As a bonus, you also get a better understanding of how the domain name system, DNS, works. When a domain (also called zone) is sent to Zonemaster, the program investigates the state of the domain from beginning to end. This is done through Zonemaster examining DNS from the root (.) to the TLD (top-level domain, for example, .se), and then finally through the DNS servers that contain information about the specified domain (for example, zonemaster.se). Zonemaster is also able to perform many other tests, for example, checking the DNSSEC signatures, that different hosts can be accessed and that the IP addresses are valid. This is all to make sure that your domain runs as good as possible",
+    "ABOUT_DNS_TEXT": "Most people who use the Internet have never reflected on what actually happens when one types an address (i.e., URL) in their web browser and then clicks the 'Enter' button. In short, the Domain Name System (DNS) is a database that associates domain names to Internet Protocol (IP) addresses, the same way a phone book associates names to phone numbers. An IP address is a unique series of numbers that identifies every computer that is connected to the Internet. It’s similar to the way every telephone has its own number in the telephone network. Such long series of numbers work fine for computers, but for people it is way more difficult to memorize. Thus, the DNS has been developed as an extra system to do this mapping transparently, for example in cases where we want to reach a web server on a specific site or an email server to send an email to.",
```

Comment:
> Most people who use the Internet have never reflected on what actually happens when one types an address (i.e., URL) in their web browser and then clicks the 'Enter' button. 

For this type of text "URL" does not add anything. On the contrary, it make it more complex.

> In short, the Domain Name System (DNS) is a database that associates domain names to Internet Protocol (IP) addresses, the same way a phone book associates names to phone numbers.

For this text "Internet Protocol" does not add anything. It is better to directly refer to IP address, better know by people.

> Thus, the DNS has been developed as an extra system to do this mapping transparently, for example in cases where we want to reach a web server on a specific site or an email server to send an email to.

Suggest:

"Thus, the DNS has been developed as an extra layer to map between names that we can remember and IP addresses used by the computers whenever we want to reach a web server on a specific site or use other services on Internet."

Reply:
I do not fully agree. More people have a picture of what "IP" is than "Internet Protocol". Actually the same is with "DNS" vs. "Domain Name System". It would be better with "About domain names and DNS".

I would be better to add "What do 'IP' and 'DNS' stand for?" as a new FAQ and there expand and explain.

Reply:
> It would be better with "About domain names and DNS".

Agreed, for a title we should use the acronym. I will update on next review.

> More people have a picture of what "IP" is than "Internet Protocol". Actually the same is with "DNS" vs. "Domain Name System".

Agreed as well, acronyms are way more well-known.

Still, in a descriptive text such as this, I feel like giving the actual definition of an acronym (which immediately follows such definition, avoiding any confusion) is particularly appropriate and costless. Having an entry in the FAQ feels unnecessary to me. Let's hear what the others think.

Reply:
@matsduf If I were to inverse the position of the acronym with its definition, would that somehow read better for you? e.g `DNS (Domain Name System)` instead of `Domain Name System (DNS)`. Another option that will soon be possible would be to give the definition of the acronym in a mouseover (hover) element.

Reply:
Yes, `DNS (Domain Name System)` is better and easier to read when (as in this case) the abbreviation is better known than the spelled-out term.

Reply:
Mouse over could be another option, but that would make handling of the translations more complext, wouldn't it? In the translated text it might not be "Domain Name System". It might be "domännamnssystem".

Reply:
Updated